advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    timestep = 900.                            # time step in seconds
</pre><p>I would generally advise to make comments on variables in docstring stile like here: https://github.com/ai2cm/pace/blob/0b26998413563aeb7cb987812c94a7b531b581b7/dsl/pace/dsl/stencil.py#L429</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>